Jenkinsfile

```diff
@@ -35,68 +35,6 @@ pipeline {
         stash name: 'src', includes: "kframework-${env.VERSION}-src.tar.gz"
```

Comment:
Was the source tar piece of this build process being primarily used as input to the the following steps? Is the rest of the Jenkinsfile still have value to keep?

Reply:
No, this is mostly some legacy thing, I don't think it's actually used anymore. Maybe in building the brew package?